README.md

```diff
@@ -36,7 +36,7 @@ prost-types = "0.12"
 ```
 
 The recommended way to add `.proto` compilation to a Cargo project is to use the
-`prost-build` library. See the [`prost-build` documentation](prost-build) for
+`prost-build` library. See the [`prost-build` documentation](https://docs.rs/prost-build/latest/prost_build/) for
```

Comment:
Could you use move the link to a footnote? This will keep the plain text more readable.

For example: https://github.com/tokio-rs/prost/blob/3cb37f2d2064d478db96b16b3d87582b14631224/README.md?plain=1#L48-L50

Reply:
@rcorre The new code doesn't have the desired behavior. I think you need square brackets `[]` at the link and the footnote. 

See the rendered version at: https://github.com/tokio-rs/prost/blob/5d6d71bb4971a82618f33835a5b69f54aa4564f9/README.md

This cheat sheet has some examples: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Reply:
Oops, sorry! Should be good now